js/views/userShort.js

```diff
@@ -0,0 +1,109 @@
+import $ from 'jquery';
+import BaseVw from './baseVw';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+import followUtils from '../utils/follow';
+import Profile from '../models/Profile';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+    this.guid = options.guid || this.model.get('guid');
+    this.ownGuid = this.guid === app.profile.id;
+    this.profileArgs = {}; // create blank for placeholder render
+    this.followedByYou = app.ownFollowing.get(this.guid) !== undefined;
+    // TODO: add in code to determine if this user is in the viewer's moderator list
+    this.ownMod = false;
+
+    this.loadUser();
+    /* the view should be rendered when it is created and before it has data, so it can occupy
+       space in the DOM while the data is being fetched. */
+  }
+
+  loadUser(guid = this.guid) {
+    let profile;
+    let profileFetch;
+
+    this.loading = true;
+
+    if (guid === app.profile.id) {
+      // don't fetch our this user's own profile, since we have it already
+      profileFetch = $.Deferred().resolve();
+      profile = app.profile;
+    } else {
+      profile = new Profile({ id: guid });
+      profileFetch = profile.fetch();
+    }
+
+    profileFetch.done(() => {
```

Comment:
Can we make this profileFetch an instance level variable (`this.profileFetch`) and in remove() abort it? Also, in the done and fail handlers, can we make those not execute if the view has been removed (i.e. make the first line `if (this.isRemoved()) return;`)?

We may have a whole lot of user views on the page and there may be some queued up / in progress fetches. If the user navigates away from the page (which will ultimately cause this view's remove() to be called) we want to cancel those fetches. Otherwise, any fetches on our new page will be queued behind these. Also, odd things can happen if the handlers execute when we're on a new page.